angular/projects/common/src/app/utils/record-constants.ts

```diff
@@ -6,6 +6,14 @@ export class EpicProjectIds {
   public static readonly coastalGaslinkId = '588511c4aaecd9001b825604';
 }
 
+export class ApplicationRoles {
+  public static readonly ADMIN       = 'sysadmin';
+  public static readonly ADMIN_NRCED = 'admin:nrced';
+  public static readonly ADMIN_LNG   = 'admin:lng';
+  public static readonly ADMIN_EPIC  = 'admin:epic';
```

Comment:
I don't believe we have a need for the role `admin:epic`.  It is an error in the bug task.

Reply:
Removed admin:epic